### PR TITLE
Added Postgres

### PR DESCRIPTION
**Pull request by theofficetroll**
I apologize for sparse information in the Commits. I had a Git error and needed to backtrack several commits so my detailed notes went away.

Updated README

Wrote API endpoints

Wrote generator

https://trello.com/c/nJINowMz

### CODE REVIEW

File: postgresdb/postgres.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+const newTable = (req, res) => {
+  pool.query('CREATE TABLE products (productId int, name varchar, gender varchar, category varchar, styleId int)', (err, data) => {
+    if (err) {
+      throw err;
+    }
+    console.log('Table created successfully');
+  })
+}
```
Comment (TandyN) [inline postgresdb/postgres.js:72]: What's the use case here?

**Review by TandyN — APPROVED** [review]
- Created routes to handle CRUD operations with Postgres queries.
- Created a data generator and exporting it as a csv file for your 1m records.

I see no conflicts.

File: postgresdb/postgres.js (hunk 1 above)
Comment (theofficetroll) [reply postgresdb/postgres.js:72]: I put this in to create the table directly, but ended up doing it through command line instead. I may end up using this later, so I think I'll leave it for now.

Comment (martinsung00) [reply postgresdb/postgres.js:72]: Although it’s a great method, a very well written one; I recommend removing it as we should try to follow the YAGNI engineering rule: https://en.m.wikipedia.org/wiki/You_aren't_gonna_need_it

File: seedGenerator.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const generator = require('./database/helpers/generator.js');
+const csvWriter = require('csv-write-stream');
+const writer = csvWriter();
+const fs = require('fs');
+// const db = require('./postgresdb/postgres.js');
```
Comment (martinsung00) [inline seedGenerator.js:5]: In a similar case to the above; if you don’t need the commented out line, it’s recommended that it’s removed. On top of making it organized, it’ll also make your PR even more clean and amazing looking; not that it’s not amazing already

Comment (theofficetroll) [reply seedGenerator.js:5]: This one I remember removing, so I'm assuming that it was part of the stuff I lost in my git issues.
Good catch :-)

**Pull request merged**